plex_set_tracks.py

```diff
@@ -188,15 +186,14 @@ def getNumFromUser(prompt):
         Parameters:
             prompt(str): The prompt the user will be given before receiving input.
     """
-    isValidNum = False
-    while not isValidNum:
+    while True:
```

Comment:
I typically like to avoid while True loops with break exit statements unless it's absolutely necessary. While it's definitely clunkier, the while condition method makes it a lot easier to see what that loop does on first glance.